tests/scalers/argo-rollouts.test.ts

```diff
@@ -2,20 +2,17 @@ import * as fs from 'fs'
 import * as sh from 'shelljs'
 import * as tmp from 'tmp'
 import test from 'ava'
-import {sleep, waitForRollout} from "./helpers";
+import {sleep} from "./helpers";
+import { PrometheusServerHelper } from './prometheus-server-helpers'
 
 const testNamespace = 'argo-rollouts-test'
 const prometheusNamespace = 'argo-monitoring'
-const prometheusDeploymentFile = 'scalers/prometheus-deployment.yaml'
 const argoRolloutsNamespace = 'argo-rollouts'
 const argoRolloutsYamlFile = tmp.fileSync()
 
 test.before(async t => {
   // install prometheus
-  sh.exec(`kubectl create namespace ${prometheusNamespace}`)
-  t.is(0, sh.exec(`kubectl apply --namespace ${prometheusNamespace} -f ${prometheusDeploymentFile}`).code, 'creating a Prometheus deployment should work.')
-  // wait for prometheus to load
-  t.is(0, waitForRollout('deployment', "prometheus-server", prometheusNamespace))
+  PrometheusServerHelper.installPrometheusServer(t, prometheusNamespace)
```

Comment:
Personal preference because I don't like `Helper` suffixes + method name is redundant given class name but not blocking
```suggestion
  PrometheusServer.install(t, prometheusNamespace)
```